### ISSUE TITLE
bug in the error banner

### ISSUE THREAD

**Issue by gabriel-alex** [sleede/fab-manager#316]
**Describe the bug**
Hello, I was trying to add a machine on FabManager and once I pushed the "Valider votre machine" button, It made appear an error message. I have an issue to solve but another issue emerged, the way the error message is printed. Each letter of the error message is in a different error banner. TL.DR. here is a an image 

![Capture](https://user-images.githubusercontent.com/25355176/148543928-a9fd2ccc-87df-413e-8074-d0a128279493.PNG)

**Server (please complete the following information):**
 - OS: Ubuntu 20.04.3 LTS
 - Kernel:  5.4.0-92-generic
 - Fab-manager version 5.3.0

**Browser (please complete the following information):**
 - Name: FireFox
 - Version: 95.0.2

**Comment by sylvainbx** [comment]
~~I can confirm the issue, it will be fixed in the next release~~
I was wrong: I was able to reproduce the issue on another page than the machine/space creation form but not on one of these forms.

**Comment by sylvainbx** [comment]
@gabriel-alex Can you please provide me a reproduction scenario of how did you fall into this issue with spaces or machines?

**Comment by gabriel-alex** [comment]
Sorry for the delay. 
I am not able to produce a reproduction scenario because I solved the issue at he origin of this screenshot. 
I have restarted docker, it has downloaded the last image without I noticed it. So one part of  FabManager have been updated but not the right bash command `\curl -sSL upgrade.fab.mn | bash -s  -- -c "rails db:seed"`. 

After doing it, it worked properly. 
Sorry.

**Comment by sylvainbx** [comment]
Ok, thanks for the feedback. I close the issue but please reopen it if you find a reproduction scenario.

**Comment by gabriel-alex** [comment]
Due to various issues while updating FabManger, I was able to reproduce the problem. My setup is quite strange, I have a main reverse proxy based on a Treafik as public server. Then I have another server that contains FabManager with its own reverse proxy. Due to a misconfiguration of the Nginx proxy, it  lead me to the following error according to logs:  
`HTTP Origin header (https://fabmanager.lf2l.fr) didn't match request.base_url (http://fabmanager.lf2l.fr)`
This error produce a "422 Unprocessable Entity" error in the navigator. Due to this error, I had the error message provided in the first post.  

To solve this issue, I added the following setting to the Nginx configuration file:
`proxy_set_header X-Forwarded-Ssl on;`

I hope this will be helpful for someone.
